### PR TITLE
{bio}[foss/2016b] MultiQC v0.9

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Requires #4447

File: easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'PythonPackage'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb]: @verdurin should this also be a `Bundle`, like it is in #3564, to make sure the required Python packages are there?

Comment (boegel) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb]: several required Python packages are not provided by the `Python` installation, e.g. `click`, `multiqc`, `PyYaml,` ...

Comment (verdurin) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb]: Are you sure they're needed? I have a local user reporting that it works fine as is i.e. it came up with the same results as he's found elsewhere.

Comment (boegel) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb]: @verdurin They'll get installed automatically even if they're not included (see EB log), but then we don't have control over which versions are being used for them...

Comment (verdurin) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb]: @boegel okay - I wondered whether the requirements had changed between versions - will add

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ca1d1627d60e7e3bed045bc74a0286c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fcdfb893c7c4af28aefbb2890cbaa84e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**